cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -2518,64 +2614,64 @@ bool CacheAllocator<CacheTrait>::moveForSlabRelease(
 
   bool isMoved = false;
   auto startTime = util::getCurrentTimeSec();
-  WriteHandle newItemHdl = allocateNewItemForOldItem(oldItem);
-
-  for (unsigned int itemMovingAttempts = 0;
-       itemMovingAttempts < config_.movingTries;
-       ++itemMovingAttempts) {
```

Comment:
why did we remove the moving attempts? In the new logic, once we marking the moving bit, the item can still have refcount > 1, right? Future readers will be blocked on a wait-context but current readers are not hindered. Wouldn't we still need this logic?